### PR TITLE
CVE-2019-14883 update

### PR DESCRIPTION
**Pull request by darunesh-RH**

### CODE REVIEW

File: 2019/14xxx/CVE-2019-14883.json [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        ]
+    },
+    "references": {
+        "reference_data": [
+            {
+                "url": "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2019-14883",
+                "name": "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2019-14883",
+                "refsource": "CONFIRM"
+            }
+        ]
```
Comment (iamleot) [inline 2019/14xxx/CVE-2019-14883.json]: Please add a reference to upstream security advisory:

 https://moodle.org/mod/forum/discuss.php?d=393586#p1586750

File: 2019/14xxx/CVE-2019-14883.json [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    },
+    "description": {
+        "description_data": [
+            {
+                "lang": "eng",
+                "value": "A vulnerability was found in Moodle 3.7 to 3.7.2 and 3.6 to 3.6.6, where tokens used to fetch inline atachments in email notifications were not disabled when a user's account was no longer active. Note: to access files, a user would need to know the file path, and their token."
```
Comment (iamleot) [inline 2019/14xxx/CVE-2019-14883.json]: Please change the description to mention fixed versions:

 3.7.3 and 3.6.7

**Comment by cve-team** [comment]
Just waiting on implementation of iamleot's comments Dhananjay and we'll get that merged. Thanks!

**Comment by cristina479** [comment]
Red Hat CNA,

Can you address the comments in this PR in order to publish this CVE ID to the CVE List?

Thank you,
Cristina

File: 2019/14xxx/CVE-2019-14883.json (hunk 1 above)
Comment (darunesh-RH) [reply 2019/14xxx/CVE-2019-14883.json]: Flaw updated with asked reference.

File: 2019/14xxx/CVE-2019-14883.json (hunk 2 above)
Comment (darunesh-RH) [reply 2019/14xxx/CVE-2019-14883.json]: Description updated with missing info.

File: 2019/14xxx/CVE-2019-14883.json [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+                "name": "https://moodle.org/mod/forum/discuss.php?d=393586#p1586750",
+                "url": "https://moodle.org/mod/forum/discuss.php?d=393586#p1586750"
+            }
+        ]
+    },
+    "description": {
+        "description_data": [
+            {
+                "lang": "eng",
+                "value": "A vulnerability was found in Moodle through version 3.6 before 3.6.7 and 3.7 to 3.7.2 before 3.7.3, where tokens used to fetch inline atachments in email notifications were not disabled when a user's account was no longer active. Note: to access files, a user would need to know the file path, and their token."
```
Comment (iamleot) [inline 2019/14xxx/CVE-2019-14883.json]: It seems there is a copypasto.
Please change from:

````
[...] 3.7 to 3.7.2 before 3.7.3 [...]
````

to:

````
[...] 3.7 before 3.7.3 [...]
````

**Pull request merged**